src/applications/financial-status-report/components/BenefitsCard.jsx

```diff
@@ -1,42 +1,70 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { fetchDebts } from '../actions';
 import Telephone, {
   CONTACTS,
   PATTERNS,
 } from '@department-of-veterans-affairs/component-library/Telephone';
 
-const BenefitsCard = () => {
+const formatter = new Intl.NumberFormat('en-US', {
+  style: 'currency',
+  currency: 'USD',
+  minimumFractionDigits: 2,
+});
+
+const BenefitCard = ({ received, total, title }) => {
   return (
-    <>
-      <div className="usa-alert background-color-only">
-        <div className="vads-u-margin-bottom--1">
-          <h4 className="vads-u-margin--0">
-            Disability compensation and pension benefits
-          </h4>
-        </div>
-        <div className="vads-u-margin-bottom--1">
-          <strong>Total amount: </strong>
-          $2,100.00
-        </div>
-        <div className="vads-u-margin-bottom--1">
-          <strong>Amount received last month: </strong>
-          $1,800.00
-        </div>
+    <div className="usa-alert background-color-only">
+      <div className="vads-u-margin-bottom--1">
+        <h4 className="vads-u-margin--0">{title}</h4>
       </div>
-      <div className="usa-alert background-color-only">
-        <div className="vads-u-margin-bottom--1">
-          <h4 className="vads-u-margin--0">Education benefits</h4>
-        </div>
-        <div className="vads-u-margin-bottom--1">
-          <strong>Total amount: </strong>
-          $1,100.00
-        </div>
-        <div className="vads-u-margin-bottom--1">
-          <strong>Amount received last month: </strong>
-          $1,100.00
-        </div>
+      <div className="vads-u-margin-bottom--1">
+        <strong>Total amount: </strong>
+        {formatter.format(parseFloat(total))}
       </div>
+      <div className="vads-u-margin-bottom--1">
+        <strong>Amount received last month: </strong>
+        {formatter.format(parseFloat(received))}
+      </div>
+    </div>
+  );
+};
+
+const Benefits = ({ income, debts, getDebts }) => {
+  useEffect(
+    () => {
+      getDebts();
+    },
+    [getDebts],
+  );
+
+  const eduDebtsTotal = debts
+    .filter(item => item.deductionCode !== '30')
```

Comment:
This is super duper nit picky but it kinda bothers me when I see a generic iteration such as `item` it reminds me a lot of Gooee code actually. Personally prefer the context of what is actually being passed so a n00b likes me knows the context of what we are sorting reducing and such.

Not required feedback just some food for thought.

Reply:
oh ya you right i thought about that and meant to back and update good catch thanks!